src/lib.rs

```diff
@@ -27,11 +176,9 @@ extern crate std;
 #[doc(hidden)]
 pub use core as __core;
 
-/// The `bitflags!` macro generates a `struct` that holds a set of C-style
-/// bitmask flags. It is useful for creating typesafe wrappers for C APIs.
+/// The macro used to generate the flag structure.
 ///
-/// The flags should only be defined for integer types, otherwise unexpected
-/// type errors may occur at compile time.
+/// See the [crate level docs](https://docs.rs/bitflags) for complete documentation.
```

Comment:
you had mentioned this aspect of it. The issue here is that this breaks locally; use a relative link instead. I believe that that means just `../index.html`, but I didn't 100% verify this